### PR TITLE
Add documentation.

### PR DESCRIPTION
**Pull request by whitingjr**
Comparing Datasets. Added content to the Horreum glossary of terms.

## Changes proposed

[x] - Comparing individual Dataset with Baseline
[x] - Comparing multiple Datasets.

- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/site/content/en/docs/Tasks/experiment-evaluation/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Then select the individual Test Run you want to compare with it's Baseline.
+
+By navigating to the uploaded run Dataset view page you will see a button  “Evaluate experiment”. Clicking this button opens a window revealing the comparison result for the current Run and the Baseline Run..
+
+{{% imgproc individual_evaluation Fit "1200x900" %}} Individual Evaluation {{% /imgproc %}}
+
+Results show the values then the percentage difference.
+
+
+Datasets Comparison View
```
Comment (johnaohara) [inline docs/site/content/en/docs/Tasks/experiment-evaluation/index.md:24]: Is this a new heading? if so, it should be highlighted as a heading
```suggestion
## Datasets Comparison View
```

File: docs/site/content/en/docs/Tasks/experiment-evaluation/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Horreum provides multiple Run comparisons in the Dataset Comparison View. We can filter based on the Filter labels defined in the Schema.
+
+Start by initially loading the Dataset list. Then click the button “Select for comparison”.
+
+{{% imgproc select_for_comparison Fit "1200x900" %}} Select for comparison {{% /imgproc %}}
+
+Next the Comparison view is displayed. This is where filters are set. Resize horizontally the window to reveal additional Filter options if obscured.
```
Comment (johnaohara) [inline docs/site/content/en/docs/Tasks/experiment-evaluation/index.md:32]: `Resize horizontally the window to reveal additional Filter options if obscured.` Is this a UI issue? 

If it is, I think we should log it as an issue and omit this sentence

**Review by johnaohara — CHANGES_REQUESTED** [review]
@whitingjr looks good, 2 minor comments. thanks a lot

File: docs/site/content/en/docs/Tasks/experiment-evaluation/index.md (hunk 2 above)
Comment (whitingjr) [reply docs/site/content/en/docs/Tasks/experiment-evaluation/index.md:32]: Yes this issue is already created. I added it here to suggest a workaround.
Will remove this.

**Comment by johnaohara** [comment]
Docs are live: https://horreum.hyperfoil.io/docs/tasks/experiment-evaluation/

**Pull request merged**